doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Usage & Billing"
+description: "Provides an overview of various factors that can affect cost."
+menu:
+  main:
+    parent: "administration"
+    weight: 10
+---
+
+From the [console](https://console.materialize.com/) (`Admin` > `Usage &
+Billing`), administrators can access their invoice. The invoice provides
+Compute and Storage usage and cost information.
+
+## Cost Factors
+
+Various factors can affect your costs, including (but not limited to):
+
+- Each [replica for a cluster](/sql/create-cluster/#credit-usage). To see the
+  number of replicas for your clusters, click on `Clusters` in the console,
+
+- Creating a [materialized view](/concepts/views/#materialized-views) as
+  materialized view creation executes the underlying query and stores the
+  results to durable storage, and materialized views incrementally updates
+  results as inputs change. To see the views in a cluster, in the console's
+  `Clusters` view, select the cluster to inspect, and click on the `Materialized
+  Views` tab.
+
+  {{< note >}}
+
+  For more information on how incremental computations can lower the cost of
+  freshness, see [How Materialized can lower the cost of freshness for data
+  team](https://materialize.com/promotions/cost-of-freshness/?utm_campaign=General&utm_source=documentation).
+
+  {{< /note >}}
+
+- Creating [indexes on a view](/concepts/indexes/) as index creation executes
+  the underlying query, and indexes maintain and, as inputs change,
+  incrementally update view results in memory.
```

Comment:
Does creating indexes increase spend outside of scenarios where you have to size up your cluster to get more memory? I'm slightly confused, because AFAIU as long as your cluster is appropriately sized, you won't pay more for creating indexes.

The same goes for sources (though here I'm actually not sure how/if we charge for storage). Maybe the confusing part is that, for anything that requires compute, you pay upfront with whatever cluster size you pick...no?